site/gatsby-site/src/components/discover/Controls.js

```diff
@@ -0,0 +1,72 @@
+import React, { useState, useEffect } from 'react';
+import REFINEMENT_LISTS from './REFINEMENT_LISTS';
+import styled from 'styled-components';
+import Stats from './Stats';
+import ClearFilters from './ClearFilters';
+import DisplayModeSwitch from './DisplayModeSwitch';
+import Filters from './Filters';
+import { Row, Col } from 'react-bootstrap';
+
+const FlexGap = styled(Col)`
+  margin: auto;
+`;
+
+const ExpandFilters = styled.button`
+  user-select: none;
+  cursor: pointer;
+  color: inherit;
+  background: none;
+  border: none;
+`;
+
+const Controls = ({ query }) => {
+  const [expandFilters, setExpandFilters] = useState(false);
+
+  const [firstRender, setFirstRender] = useState(true);
+
+  useEffect(() => {
+    if (firstRender) {
+      let filter = false;
+
+      for (let refinement of REFINEMENT_LISTS) {
+        if (query[refinement.attribute]) {
+          filter = true;
+          break;
+        }
+      }
+      if (filter) {
+        document.querySelector('#expand-filters').click();
```

Comment:
Direct DOM manipulation is a big no-no! Worst case use a ref, but here seems like you would want to use a state flag.

Reply:
I've changed it to use `setExpandFilters(true)`.